### PR TITLE
Update vibi_tables.sql

### PR DESCRIPTION
**Pull request by jreinier**

### CODE REVIEW

**Comment by jreinier** [comment]
My apologies, I missed new mapping opportunities when I commented yesterday.  Plot number, species and voucher number should now be mapped from ENTER FDS1 and ENTER FDS2 tabs in the spreadsheet to the new fds1_species_misc_info and fds2_species_misc_info tables in the database.

File: backend/mapper/conf/vibi_tables.sql [hunk 1]
```diff
@@ -218,24 +223,20 @@ INSERT INTO oh_status VALUES ('adventive'), ('cryptogeni'), ('native');
   CONSTRAINT species_pkey PRIMARY KEY (scientific_name)
 );
 -- species table will be populated from most recent version of the Ohio EPA-maintained list
---COPY species FROM 'H:/G_drive_Nat_Res_Reinier/Projects/FQAI_updates/OEPA_species_table_12_4_2015.csv' DELIMITER ',' CSV HEADER;
+COPY species FROM 'H:/G_drive_Nat_Res_Reinier/Projects/FQAI_updates/OEPA_species_table_12_4_2015.csv' DELIMITER ',' CSV HEADER;
```
Comment (nmco) [inline backend/mapper/conf/vibi_tables.sql]: this one should be commented I guess :)

File: backend/mapper/conf/vibi_tables.sql [hunk 2]
```diff
@@ -245,27 +246,35 @@ CREATE TABLE plot (
...
+  landscape_position text,
+  inland_landform text,
+  water_flow_path text,
+  llww_modifiers text,
+  llww_modifiers_other text,
   Landform_Type text references landform_type(landform_type),
+  landform_type_other text,
   Homogeneity text references homogeneity(homogeneity),
   Stand_size text references stand_size(stand_size),
   Drainage text references drainage(drainage),
```
Comment (nmco) [inline backend/mapper/conf/vibi_tables.sql]: I'm not sure how to map some of the new columns, like llww_modifiers for example. Could you add some info about the mapping ?

File: backend/mapper/conf/vibi_tables.sql [hunk 3]
```diff
@@ -309,9 +318,20 @@ CREATE TABLE plot_module_herbaceous_info (
...
+species text references species(scientific_name),
+plot_no text references plot(plot_no) ON UPDATE CASCADE ON DELETE CASCADE,
+module_id int4 references module(module_id),
+voucher_no text,
+comment text,
+browse_intensity text,
+percent_flowering text,
+percent_fruiting text
+);
+
```
Comment (nmco) [inline backend/mapper/conf/vibi_tables.sql]: I don't know how to map columns browse_intensity, percent_flowering and percent_fruiting. I suppose that column comment should be mapped with notes on the spreadsheet ?

File: backend/mapper/conf/vibi_tables.sql [hunk 4]
```diff
@@ -320,6 +340,17 @@ CREATE TABLE plot_module_woody_raw (
...
```
Comment (nmco) [inline backend/mapper/conf/vibi_tables.sql]: I don't know how to map columns browse_intensity, percent_flowering and percent_fruiting. I suppose that column comment should be mapped with notes on the spreadsheet ? This is the last mapping question :)

File: backend/mapper/conf/vibi_tables.sql (hunk 1 above)
Comment (jreinier) [inline backend/mapper/conf/vibi_tables.sql]: New table leap_landcover_classification and its associated INSERT are for new data that will be collected going forward using a data collection process other than the spreadsheet.  There should be no mapping to the spreadsheet.  But we would like this table to be visible through the web page.

File: backend/mapper/conf/vibi_tables.sql (hunk 2 above)
Comment (jreinier) [inline backend/mapper/conf/vibi_tables.sql]: None of the new columns above have equivalents in the spreadsheet.  They were added for new data that will be collected going forward using a data collection process other than the spreadsheets.  Therefore, there should be no mapping between the spreadsheet and these new columns.  However, we do want these new columns visible through the web page.

File: backend/mapper/conf/vibi_tables.sql (hunk 3 above)
Comment (jreinier) [inline backend/mapper/conf/vibi_tables.sql]: Browse intensity can be mapped to column K ("B") in the ENTER FDS1 tab of the spreadsheet.  Percent_flowering and percent_fruiting do not have equivalents in the spreadsheet so there should be no mapping.  Yes, column comment should be mapped to notes/comments in the spreadsheet.

File: backend/mapper/conf/vibi_tables.sql (hunk 4 above)
Comment (jreinier) [inline backend/mapper/conf/vibi_tables.sql]: Follow the same guidance as above.  Browse_intensity gets mapped to column K ("Browsed stems") of ENTER FDS2 tab.

**Pull request merged**